Microsoft.Bot.Builder.Ai.LanguageGeneration/Resolver/ILGEndpoint.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.Builder.Ai.LanguageGeneration.Resolver
+{
+    public interface ILGEndpoint
+    {
+        string EndpointKey { get; }
```

Comment:
Is this the subscription key? If yes, I suggest renaming to follow the same convention of Bot Framework or Cognitive Services.